paddle/fluid/operators/fetch_barrier_op.cc

```diff
@@ -43,7 +43,7 @@ class FetchBarrierOp : public framework::OperatorBase {
     // For profiling
     platform::RecordEvent record_event(Type(), &ctx);
 
-    auto rpc_client = detail::RPCClient::GetInstance();
+    auto rpc_client = detail::GRPCClient::GetInstance();
```

Comment:
To use the client, better to use the type of base class, since we use a singleton, you can make the base class RPCClient as a singleton, and other clients can inherit from it.

Reply:
Done. Thanks!